packages/webapp/__tests__/NotificationsPage.tsx

```diff
@@ -108,6 +108,11 @@ it('should show the welcome notification', async () => {
 });
 
 it('should not show the welcome notification if we are not at the last page', async () => {
+  const temp = globalThis.window.Notification;
```

Comment:
I had to manually make the notifications available here even though they are not available in the testing environment